esti/s3_gateway_test.go

```diff
@@ -329,6 +332,40 @@ func TestS3HeadBucket(t *testing.T) {
 	})
 }
 
+// getOrCreatePathToLargeObject returns a configured existing large
+// (largeDataContentLength, 6MiB) object, or creates a new one under
+// testPrefix.
+func getOrCreatePathToLargeObject(t *testing.T, ctx context.Context, s3lakefsClient *minio.Client, repo, branch string) (string, int64) {
+	t.Helper()
+
+	path := "source-file"
+	s3Path := fmt.Sprintf("%s/source-file", branch)
+
+	if physicalAddress := viper.GetString("large_object_path"); physicalAddress != "" {
+		res, err := client.LinkPhysicalAddressWithResponse(ctx, repo, branch, &apigen.LinkPhysicalAddressParams{Path: path}, apigen.LinkPhysicalAddressJSONRequestBody{
+			Checksum: "dont-care",
+			// TODO(ariels): Check actual length of object!
```

Comment:
Do we need that before merge?

Reply:
No. It might improve ergonomics of creating a new object, but is not critical.